compiler/defer.go

```diff
@@ -151,9 +155,87 @@ func (b *builder) createDefer(instr *ssa.Defer) {
 		values = append(values, context)
 		valueTypes = append(valueTypes, context.Type())
 
+	} else if  builtin, ok := instr.Call.Value.(*ssa.Builtin); ok {
+		var funcName string
+		switch builtin.Name() {
+		case "close":
+			funcName = "chanClose"
+		default:
+			b.addError(instr.Pos(), fmt.Sprint("TODO: Implement defer for ", builtin.Name()))
```

Comment:
No need for `fmt.Sprint`, you can simply add the strings together.
```suggestion
			b.addError(instr.Pos(), "todo: Implement defer for " + builtin.Name())
```